### ISSUE TITLE
[Feature] global affinity and tolerations

### ISSUE THREAD

**Issue by fauzanelka** [kubecost/cost-analyzer-helm-chart#3439]
### Problem Statement

It is such a pain to declare affinity to all services manually, can't we have `global.affinity` and `global.tolerations` instead?
This will override all affinities and tolerations declared (if they exists) on the services.

### Solution Description

Add `global.affinity` and `global.tolerations` to helm chart.
Update the conditions of affinity and tolerations on the helm template to check `global.affinity` and `global.tolerations` values.

### Troubleshooting

- [X] I have read and followed the [issue guidelines](https://github.com/kubecost/cost-analyzer-helm-chart/blob/develop/ISSUE_GUIDELINES.md) and this is a feature request only for the Helm chart.
- [X] I have searched other issues in this repository and mine is not recorded.

**Comment by chipzoller** [comment]
Agree, would be a useful thing. Is this an area where you'd like to contribute?
